src/scripts/servinit-defaults

```diff
@@ -4,7 +4,7 @@
 # doc/examples/servexec.cfg
 #
 # It reads all sv_* variables in doc/examples/servexec.cfg, extracts their
-# default value by executing them in a Red Eclipse Legacy server, then feeds
+# default value by executing them in a Blue Nebula server, then feeds
```

Comment:
See line 16

Reply:
The script is broken anyway, since we don't have these scripts any more. It should be fixed when ever needed again to extract the variables, I guess.